js/ui/notify.d.ts

```diff
@@ -1,7 +1,45 @@
+type NotifyStackPosition = 'top left' | 'top right' | 'bottom left' | 'bottom right' | 'top center' | 'bottom center' | 'left center' | 'right center' | 'center';
+type NotifyStackDirection = 'down' | 'up' | 'left' | 'right' | 'down-reverse' | 'up-reverse' | 'left-reverse' | 'right-reverse';
+
+interface StackOptions {
```

Comment:
Is it possible to avoid using `Options` word?

Reply:
it's the same name as we'll have in our docs. If it's ok to use different name for interface in declaration?